packages/rx_bloc_cli/lib/src/commands/create_command.dart

```diff
@@ -70,6 +76,7 @@ class CreateCommand extends Command<int> {
   final _counterString = 'enable-feature-counter';
   final _deepLinkString = 'enable-feature-deeplinks';
   final _widgetToolkitString = 'enable-feature-widget-toolkit';
+  final _facebookAuthString = 'enable-facebook-auth';
```

Comment:
This should be renamed to `enable-feature-facebook-auth` to match the rest of the optional features.